src/cli.js

```diff
@@ -155,7 +161,9 @@ yargs
         passthroughCustomScalars: argv["passthrough-custom-scalars"] || argv["custom-scalars-prefix"] !== '',
         customScalarsPrefix: argv["custom-scalars-prefix"] || '',
         addTypename: argv["add-typename"],
-        namespace: argv.namespace
+        namespace: argv.namespace,
+        operationIdsPath: argv["operation-ids-path"],
+        generateOperationIds: !!argv["operation-ids-path"]
```

Comment:
I do think having this as a separate option reads a lot better, especially in `codeGeneration.js`, which doesn't need to know or care about file paths.